snap/snapcraft.yaml

```diff
@@ -294,18 +260,18 @@ parts:
         ;;
 
         *)
-          branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.branch)"
-          url="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.url)"
-          commit="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.commit)"
-          patch_dir="${SNAPCRAFT_STAGE}/qemu-lite/patches/"
+          branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu.tag)"
+          url="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu.url)"
+          patch_dir="${SNAPCRAFT_STAGE}/qemu/patches/4.0.x"
```

Comment:
Is there any way (or advantage) to creating a variable for `4.0.x`?

Reply:
may be not, 4.0.x is only used here